v3/lints/rfc/lint_name_constraint_not_fqdn.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package rfc
+
+import (
+	"fmt"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+type nameConstraintNotFQDN struct{}
+
+func (l *nameConstraintNotFQDN) Initialize() error {
+	return nil
+}
+
+func (l *nameConstraintNotFQDN) CheckApplies(c *x509.Certificate) bool {
+	return util.IsExtInCert(c, util.NameConstOID)
+}
+
+func (l *nameConstraintNotFQDN) Execute(c *x509.Certificate) *lint.LintResult {
+	for _, subtreeString := range c.PermittedURIs {
+		var host = subtreeString.Data
+		if string(host[0]) == string(".") {
+			host = "l" + host
```

Comment:
Why prefix the host with "l" here? Would it be better to strip the leading "." instead? I think that might be less confusing to the reader. 

It also matches the approach `util.IsFQDN` takes for wildcards/question marks.